plugins/pa11y/readme.md

```diff
@@ -0,0 +1,81 @@
+# Description 
+ Pa11y is your automated accessibility testing pal. It runs accessibility tests on your pages via the command line or Node.js, so you can automate your testing process.
```

Comment:
**suggestion**: we should go into more detail how Pa11y integrates with Tool Kit here